addons/point_of_sale/static/src/css/pos.css

```diff
@@ -1266,6 +1266,23 @@ td {
     padding-top:15px;
 }
 
+/*  ********* The BlockUI Component (frontend)  ********* */
+.o_blockUI {
+    position: fixed;
```

Comment:
Debatable, but grouping rules it's ok for me.  Let's just order them according to our guidelines (https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines#generic-rules).